### PR TITLE
Fabo/show boot error

### PR DESCRIPTION
**Pull request by faboweb**
Shows an error if there is an error in the bootprocess to avoid a spinner of death

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f3ad1b8818927a5e70b257e--lunieio.netlify.app

File: app/src/initializeApp.js [hunk 1]
```diff
@@ -37,7 +37,16 @@ function setOptions(urlParams, store) {
   }
 }
 
+export const bootError = (error) => {
+  document.querySelector("#app").innerText = `
+    Lunie failed to start. Please contact us at http://help.lunie.io/
```
Comment (jbibla) [inline app/src/initializeApp.js:42]: i'm not sure if the help center is the right place to send people.

Comment (faboweb) [reply app/src/initializeApp.js:42]: where should we send them? email?

Comment (jbibla) [reply app/src/initializeApp.js:42]: would be nice to have a link that opens intercom automagically - but i guess until then we don't have to send them anywere. yeah we can add a support email.

Comment (faboweb) [reply app/src/initializeApp.js:42]: Help Center has Intercom

**Comment by faboweb** [comment]
@jbibla can you some basic styles in a free minute?

File: app/src/initializeApp.js (hunk 1 above)
Comment (jbibla) [reply app/src/initializeApp.js:42]: it's not really obvious / in the corner - but it's ok - this is a rare screen 😄

File: app/src/initializeApp.js [hunk 2]
```diff
@@ -37,7 +37,16 @@ function setOptions(urlParams, store) {
   }
 }
 
+export const bootError = (error) => {
+  document.querySelector("#app").innerText = `
```
Comment (jbibla) [inline app/src/initializeApp.js:41]: is this really the best way to do this?

File: app/src/initializeApp.js (hunk 1 above)
Comment (jbibla) [inline app/src/initializeApp.js:42]:
```suggestion
    Uh oh. Lunie failed to start. Our team has been notified of the issue. Please try refreshing this page in a few minutes. 
```

what about something like this?

**Comment by jbibla** [comment]
> @jbibla can you some basic styles in a free minute?

yeah - but it would be a lot simpler if it had a div or class. does it need to replace the whole app instead of using one of our existing error components?

File: app/src/initializeApp.js (hunk 2 above)
Comment (faboweb) [reply app/src/initializeApp.js:41]: need to do basic JS here as the whole Vue app might have been crashed so we can't use virtual dom rendering.

File: app/src/initializeApp.js (hunk 1 above)
Comment (faboweb) [reply app/src/initializeApp.js:42]: You don't want to give a way to contact us?

**Comment by faboweb** [comment]
> yeah - but it would be a lot simpler if it had a div or class. does it need to replace the whole app instead of using one of our existing error components?

we can add a class to `#app` when calling the error function. or we add the error in another element. Overwriting the content of `#app` makes sure we don't display broken content and only show the error.

**Comment by faboweb** [comment]
image

**Comment by Bitcoinera** [comment]
Is this one ready to review?

**Pull request merged**